### PR TITLE
mmclassification ConformerHead support

### PR DESCRIPTION
**Pull request by xizi**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by xizi** [comment]
> [CLA assistant check](https://cla-assistant.io/open-mmlab/mmdeploy?pullRequest=1905) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/open-mmlab/mmdeploy?pullRequest=1905) before we can accept your contribution.
> 
> **lishengxi** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/open-mmlab/mmdeploy?pullRequest=1905) it.

The CLA agreement has been completed

**Comment by lvhan028** [comment]
May provide a test command

**Review by RunningLeon — APPROVED** [review]
LGTM

File: mmdeploy/codebase/mmcls/models/classifiers/base.py [hunk 1]
```diff
@@ -35,9 +35,11 @@ def base_classifier__forward(
     if self.head is not None:
         output = self.head(output)
 
-    from mmcls.models.heads import MultiLabelClsHead
+    from mmcls.models.heads import ConformerHead, MultiLabelClsHead
```
Comment (RunningLeon) [inline mmdeploy/codebase/mmcls/models/classifiers/base.py:38]: Could we add test for `conformer` in https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/tests/regression/mmcls.yml?

**Review by irexyc — APPROVED** [review]

**Pull request merged**